### PR TITLE
MM-T1535 Team setting / Invite code text

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary

- Added e2e test to verify if help text under team invite is present

#### Ticket Link

https://automation-test-cases.vercel.app/test/MM-T1535

#### Related Pull Requests

none

#### Screenshots

n/a

### CODE REVIEW

File: e2e/cypress/integration/team_settings/teams_spec.js [hunk 1]
```diff
@@ -189,4 +189,22 @@ describe('Teams Suite', () => {
         // * Ensure user is logged out
         cy.url({timeout: TIMEOUTS.HALF_MIN}).should('include', 'login');
     });
+
+    it.only('MM-T1535 Team setting / Invite code text', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/team_settings/teams_spec.js:193]:
```suggestion
    it('MM-T1535 Team setting / Invite code text', () => {
```

File: e2e/cypress/integration/team_settings/teams_spec.js [hunk 2]
```diff
@@ -189,4 +189,22 @@ describe('Teams Suite', () => {
...
+        // # Login as Admin and visit /
+        cy.apiAdminLogin();
```
Comment (saturninoabril) [inline e2e/cypress/integration/team_settings/teams_spec.js:195]: This is redundant. It's already logged in from beforeEach.

File: e2e/cypress/integration/team_settings/teams_spec.js [hunk 3]
```diff
@@ -189,4 +189,22 @@ describe('Teams Suite', () => {
...
+        cy.findByLabelText('main menu').click();
+
+        // # Click on team settings menu item
+        cy.findByText('Team Settings').should('exist').click();
+
+        // # Open edit settings for invite code
+        cy.findByText('Invite Code').should('exist').click();
+
+        // * Verify invite code help text is visible
+        cy.findByText('The Invite Code is part of the unique team invitation link which is sent to members you’re inviting to this team. Regenerating the code creates a new invitation link and invalidates the previous link.');
```
Comment (saturninoabril) [inline e2e/cypress/integration/team_settings/teams_spec.js:208]: Please add assertion, like `should('be.visible')`, and probably need to scroll into view before it.

File: e2e/cypress/integration/team_settings/teams_spec.js [hunk 4]
```diff
@@ -189,4 +189,22 @@ describe('Teams Suite', () => {
...
         cy.url({timeout: TIMEOUTS.HALF_MIN}).should('include', 'login');
     });
+
+    it.only('MM-T1535 Team setting / Invite code text', () => {
+        // # Login as Admin and visit /
+        cy.apiAdminLogin();
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the hamburger menu
+        cy.findByLabelText('main menu').click();
```
Comment (josephbaylon) [inline e2e/cypress/integration/team_settings/teams_spec.js:199]:
```suggestion
        cy.findByLabelText('main menu').should('be.visible').click();
```

File: e2e/cypress/integration/team_settings/teams_spec.js [hunk 5]
```diff
@@ -189,4 +189,22 @@ describe('Teams Suite', () => {
...
+
+        // # Click on team settings menu item
+        cy.findByText('Team Settings').should('exist').click();
```
Comment (josephbaylon) [inline e2e/cypress/integration/team_settings/teams_spec.js:202]:
```suggestion
        cy.findByText('Team Settings').should('be.visible').click();
```

File: e2e/cypress/integration/team_settings/teams_spec.js [hunk 6]
```diff
@@ -189,4 +189,22 @@ describe('Teams Suite', () => {
...
+
+        // # Open edit settings for invite code
+        cy.findByText('Invite Code').should('exist').click();
```
Comment (josephbaylon) [inline e2e/cypress/integration/team_settings/teams_spec.js:205]:
```suggestion
        cy.findByText('Invite Code').should('be.visible').click();
```

**Pull request merged**